crates/biome_js_analyze/tests/specs/nursery/noImplicitAnyLet/valid.ts

```diff
@@ -0,0 +1,6 @@
+/* should not generate diagnostics */
+
```

Comment:
We should add `const`, [`using`](https://github.com/tc39/proposal-explicit-resource-management) and `for-of` examples to ensure that they are ignored.

e.g.

```ts
const x = 0;
for(let y of xs) {}
using z = f();
```